test/core/test_api_versions.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const test = require('tape');
+const runner = require('../../core/lib/runner').runner;
+
+test('api versions', function(t) {
```

Comment:
rename test file to test_additionalInfo and also this line to
```suggestion
test('additionalInfo', function(t) {
```